icu4j/perf-tests/converterperf.pl

```diff
@@ -16,7 +16,13 @@
 # Test class
 my $TESTCLASS = 'com.ibm.icu.dev.test.perf.ConverterPerformanceTest';
 
-my $CLASSES = './out/bin:../tools/misc/out/bin/:../icu4j.jar:../icu4j-charset.jar';
+my $OS=$^O;
+my $CLASSPATH;
+if ($OS eq "linux" || $OS eq "darwin") {
+	$CLASSPATH = './target/*:./target/dependency/*';
+} else {
+	$CLASSPATH = './target/*;./target/dependency/*';
+}
```

Comment:
I would flip the condition to make the Unix/Linux style `CLASSPATH` the else case, since Windows is the exception to the norm.

Reply:
Done.

Also added the report files to `.gitignore` (for example `perf May  3 151550.html`)